cmd/export_accounts.go

```diff
@@ -36,6 +38,7 @@ the export_ledger_entry_changes command.`,
 		}
 
 		failures := 0
+		numBytes := 0
```

Comment:
❓ jw is numBytes the most helpful way to see progress? Or would `numOfAccounts` or something like that be more helpful. bytes is less clear to me but up to you

Reply:
The [printTransformStats](https://github.com/stellar/stellar-etl/blob/master/cmd/export_ledgers.go#L51-L54) function already reports the number of "transforms" (in this case, should be accounts) because it is the number of unique records in the file. 

I added `numBytes` so that we know how large the file is that is written to local storage. The reason I did this is the following Airflow task reports how large the file is that is uploaded to GCS. When debugging, we can compare the reported file sizes as a check to see if the file we uploaded to GCS is the same size as the file written to the local volume in the Docker container.